docs/utils.md

```diff
@@ -77,7 +77,7 @@ magic.util.rsaKeypairGen()
 
 ### magic.util.timingSafeCompare
 
-Implements a timing safe comparision between two strings. The comparision is completed using the timing safe buffer comparision using `crypto` module (falling back to `libsodium` if `crypto` is not available) and string length checks.
+Implements a timing safe comparison between two strings. The comparison is completed using string length checks and the timing safe buffer comparison in the `crypto` module (falling back to `libsodium` if `crypto` is not available).
```

Comment:
This line is fine, but [line 6](https://github.com/auth0/magic/pull/60/files#diff-9737518c28fc649aca7ba87162be1d12R6) has `comparision`, should be `comparison`.